_includes/head.html

```diff
@@ -33,14 +33,14 @@
     {%- endif -%}">
     <meta property="og:url" content="{{- page.url | absolute_url -}}">
     <link rel="canonical" href="{{- page.url | absolute_url -}}" />
-    
+
     {%- if site.favicon -%}
         <link rel="shortcut icon" href="{{- site.baseurl- }}{{- site.favicon -}}" type="image/x-icon">
     {%- else -%}
         <link rel="shortcut icon" href="{{- site.baseurl -}}/assets/img/favicon.ico" type="image/x-icon">
     {%- endif -%}
-    
-    <link rel="stylesheet" href="{{- site.baseurl -}}/assets/scss/styles.css">
+
+    <link rel="stylesheet" href="{{- site.baseurl -}}/assets/css/styles.css">
```

Comment:
did we change the dir path?

Reply:
Yup, `styles.scss` was moved into the `css` folder earlier in #307.